src/Tooltip/index.tsx

```diff
@@ -7,6 +7,7 @@ import Arrow from './Arrow'
 import TipContainer from './TipContainer'
 
 import styles from './tooltip.module.scss'
+import { Fade } from '..'
```

Comment:
Please use the Fade component directly - '../Fade'. If you want to see why, run `yarn build`, then look at the Tooltip file in the dist folder. It's going to include every component because you're importing from the index file which contains all components. So essentially, this breaks the treeshaking

Reply:
ah good catch